src/Components/Web/src/Forms/InputSelect.cs

```diff
@@ -24,16 +27,51 @@ public class InputSelect<[DynamicallyAccessedMembers(DynamicallyAccessedMemberTy
         /// </summary>
         [DisallowNull] public ElementReference? Element { get; protected set; }
 
+        /// <summary>
+        /// Gets or sets the current value of the input as a string array.
+        /// </summary>
+        protected string?[]? CurrentValueAsStringArray
+        {
+            get
+            {
+                if (CurrentValue is not Array array)
+                {
+                    return null;
+                }
+                
+                return array
+                    .Cast<object?>()
+                    .Select(item => item?.ToString())
+                    .ToArray();
```

Comment:
If we were keeping this getter, it could be made less allocatey by using a `StringBuilder` and directly calling `ToString` on each array entry without first potentially boxing as `object`.

However if possible it would be better still to remove the getter since I think we don't need it (see other comments).